frontends/election-manager/src/screens/print_test_deck_screen.test.tsx

```diff
@@ -47,15 +119,218 @@ test('Printing all L&A packages sorts precincts', async () => {
+  const electionWithLegalSizeHmpbsDefinition = _.cloneDeep(
+    electionWithMsEitherNeitherDefinition
+  );
+  electionWithLegalSizeHmpbsDefinition.election.ballotLayout!.paperSize =
+    BallotPaperSize.Legal;
+
+  renderInAppContext(<PrintTestDeckScreen />, {
+    electionDefinition: electionWithLegalSizeHmpbsDefinition,
+    logger: mockLogger,
+    printBallotRef: mockPrintBallotRef,
+    printer: mockPrinter,
+  });
+
+  userEvent.click(screen.getByText('District 5'));
+
+  const printText = 'Printing L&A Package: District 5';
+  await screen.findByText(printText);
+  await waitFor(() => expect(mockPrinter.print).toHaveBeenCalledTimes(1));
+  expect(mockPrinter.print).toHaveBeenLastCalledWith({ sides: 'one-sided' });
+  await waitFor(() =>
+    expect(mockKiosk.log).toHaveBeenLastCalledWith(
+      expect.stringContaining(LogEventId.TestDeckTallyReportPrinted)
+    )
+  );
+  jest.advanceTimersByTime(ONE_SIDED_PAGE_PRINT_TIME_MS + 1);
+
+  await screen.findByText(printText);
+  await waitFor(() => expect(mockPrinter.print).toHaveBeenCalledTimes(2));
+  expect(mockPrinter.print).toHaveBeenLastCalledWith({ sides: 'one-sided' });
+  await waitFor(() =>
+    expect(mockKiosk.log).toHaveBeenLastCalledWith(
+      expect.stringContaining(LogEventId.TestDeckPrinted)
+    )
+  );
+  expect(mockKiosk.log).toHaveBeenLastCalledWith(
+    expect.stringContaining('BMD paper ballot test deck printed')
+  );
+  jest.advanceTimersByTime(3 * ONE_SIDED_PAGE_PRINT_TIME_MS + 1);
+
+  userEvent.click(
+    await screen.findByText('Legal Paper Loaded. Continue Printing')
+  );
+
+  await screen.findByText(printText);
+  await waitFor(() => expect(mockPrinter.print).toHaveBeenCalledTimes(3));
+  expect(mockPrinter.print).toHaveBeenLastCalledWith({
+    sides: 'two-sided-long-edge',
+  });
+  await waitFor(() =>
+    expect(mockKiosk.log).toHaveBeenLastCalledWith(
+      expect.stringContaining(LogEventId.TestDeckPrinted)
+    )
+  );
+  expect(mockKiosk.log).toHaveBeenLastCalledWith(
+    expect.stringContaining('Hand-marked paper ballot test deck printed')
+  );
+  jest.advanceTimersByTime(LAST_PRINT_JOB_SLEEP_MS + 1);
+
+  await screen.findByText('District 5');
+  expect(screen.queryByText('Printing')).not.toBeInTheDocument();
+});
+
+test('Printing L&A packages for all precincts, when HMPBs are not letter-size', async () => {
+  const electionWithLegalSizeHmpbsDefinition = _.cloneDeep(
+    electionWithMsEitherNeitherDefinition
+  );
+  electionWithLegalSizeHmpbsDefinition.election.ballotLayout!.paperSize =
+    BallotPaperSize.Legal;
+
+  renderInAppContext(<PrintTestDeckScreen />, {
+    electionDefinition: electionWithLegalSizeHmpbsDefinition,
+    logger: mockLogger,
+    printBallotRef: mockPrintBallotRef,
+    printer: mockPrinter,
+  });
+
+  userEvent.click(screen.getByText('All Precincts'));
+
+  // Check that the printing modals appear in alphabetical order
+  const precinctsInAlphabeticalOrder = [
```

Comment:
Is it useful to hardcode this list? I think I asked about this in another PR but don't recall the resolution of that.

Reply:
Ah yes, I left my rationale in this comment: https://github.com/votingworks/vxsuite/pull/1863#discussion_r880750027

That said, I'm beginning to question that rationale a bit as I've had to mimic other pieces of the logic in the original code in the tests

Reply:
Adding some extra color to the above, I once got bit by running `.sort()` on some numbers and using the same `.sort()` in tests, masking the fact that `.sort()` on numbers sorts the numbers as strings, e.g. `[1, 10, 11, 12, 2, 3, 4, 5, 6, 7, 8, 9]` (which was not what I wanted :smile:)

So I may be over-rotating on that past experience. I don't feel strongly and am happy to remove the hard-coded list if you think that's preferable